activities.el

```diff
@@ -479,12 +479,47 @@ It will not be recoverable."
   "Automatically save activities when Emacs has been idle this many seconds."
   :type 'natnum)
 
+(defvar activities-mode-map (make-sparse-keymap)
+  "The mode keymap for `activities-mode'.")
```

Comment:
Strictly speaking, I'm not sure if this is necessary, or actually correct, because it's not necessary to use `activities-mode` to use the various commands, so I'm not sure if the menu bar entries should be tied to a mode/map.

Reply:
We need the mode's keymap for the menu to appear on the menu bar. If the mode isn't active, the menu won't appear so no harm. It doesn't affect general operation.